### DISCUSSION TITLE
How much disk space for WIS/WAS

### DISCUSSION THREAD

**Discussion by rarroyo6** [toverainc/willow-inference-server#142] [Q&A] [answered]
How much disk space is needed for WIS and WAS?

**Comment by nikito** [comment, marked as answer]
From what i see of my usage, it's around 70gb total (but i run some dev stuff 🙂 so if you are purely using the docker containers it's likely less than that).  Most of the usage would be from WIS due to the large models needed for TTS inference. Note that presently WIS must be built on the machine, which can use a big chunk of space as well, but that can be reclaimed later using docker system prune.

**Reply by rarroyo6** [reply]
Thank you.
